telco-core/configuration/reference-crs-kube-compare/required/networking/NMStateNS.yaml

```diff
@@ -4,3 +4,6 @@ metadata:
   name: openshift-nmstate
   annotations:
     workload.openshift.io/allowed: management
+  labels:
+    name: openshift-nmstate
+    openshift.io/cluster-monitoring: "true"
```

Comment:
The name appears to be automatically added (though I'm not sure why...) . The monitoring looks to be a specific opt-in. If that is the case it should either be included in the reference CR as well or ignored via metadata.yaml if we don't care.

Reply:
We need someone from nmstate or networking here, I have no idea.